### DISCUSSION TITLE
Can't get out of the SEB

### DISCUSSION THREAD

**Discussion by HKPalihakkara** [SafeExamBrowser/seb-mac#316] [Q&A]
3 months ago I faced a exam using SEB in my mac and after the exam I forgot to reset the browser. Now after 3 months I accidentally open the SEB and now I can't get out of it without the quit password. But I don't have the password. 
I tried to restart the mac to get out of the browser but it still didn't work. 
Is there any way to quit SEB?

SEB version 3.2.5
Macbook pro M1 chip 

![Uploading 20230724_221836.jpg…]()

**Comment by danschlet** [comment]
Just check our [FAQ](https://safeexambrowser.org/faq/faq_en.html#MacOSX).
